src/api/constants/declarations/versions/index.ts

```diff
@@ -1,27 +1,28 @@
 import { ApplicationDeclarationVersions } from '../../../types';
 
+const VERSION_1: ApplicationDeclarationVersions = {
+  ANTI_BRIBERY: '1',
+  ANTI_BRIBERY_CODE_OF_CONDUCT: '1',
+  ANTI_BRIBERY_EXPORTING_WITH_CODE_OF_CONDUCT: '1',
+  CONFIDENTIALITY: '1',
+  CONFIRMATION_AND_ACKNOWLEDGEMENTS: '1',
+  HOW_YOUR_DATA_WILL_BE_USED: '1',
+};
+
+const VERSION_2: ApplicationDeclarationVersions = {
+  ANTI_BRIBERY: '2',
+  ANTI_BRIBERY_CODE_OF_CONDUCT: '2',
+  ANTI_BRIBERY_EXPORTING_WITH_CODE_OF_CONDUCT: '1',
+  CONFIDENTIALITY: '1',
+  CONFIRMATION_AND_ACKNOWLEDGEMENTS: '1',
```

Comment:
are the new ones not included in this file?

Reply:
No, these are the top level declaration versions. The new child declaration versions for modern slavery are in `DECLARATION_MODERN_SLAVERY_VERSIONS`